atomic_reactor/plugins/pre_resolve_remote_source.py

```diff
@@ -63,6 +65,22 @@ def json_filename(cls, name: Optional[str]):
             return REMOTE_SOURCE_JSON_FILENAME
 
 
+def transform_results(results):
+    if not results or not isinstance(results, list):
+        return
+
+    remote_sources_annotations = [
+        {"name": remote_source["name"], "url": remote_source["url"]}
+        for remote_source in results
+    ]
+
+    if not remote_sources_annotations:
+        return
+
+    return json.dumps(remote_sources_annotations)
```

Comment:
IMO we don't want to JSON-dump yet. The annotations returned from plugins should be Python objects (but compatible with JSON), the dumping to JSON happens automatically in the WorkflowData code and in the store_metadata plugin